src/model/student.js

```diff
@@ -28,13 +28,13 @@ const studentSchema = new mongoose.Schema({
   },
   password:{
       type: String,
-      required: [false, "Please provide password"],
+      required: [true, "Please provide password"],
```

Comment:
This will be false now, since Google OAuth wont consist password entry.

Reply:
Resolved